modules/serverless-creating-subscriptions-kn.adoc

```diff
@@ -17,16 +17,20 @@ To create subscriptions using the `kn` CLI, ensure that:
 
 .Procedure
 
-* Create a subscription to connect a sink to a channel.
+* Create a subscription to connect a sink to a channel:
 +
 [source,terminal]
 ----
 $ kn subscription create <subscription_name> \
-  --channel <Group:Version:Kind>:<channel_name> \
-  --sink <sink_prefix>:<sink_name> \
-  --sink-reply <sink_prefix>:<sink_name> \
-  --sink-dead-letter <sink_prefix>:<sink_name>
-----
+  --channel <Group:Version:Kind>:<channel_name> \ <1>
+  --sink <sink_prefix>:<sink_name> \ <2>
+----
+<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default channel that is backed by the Channel resource, you must prefix the channel name with the `<Group:Version:Kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for a Kafka backed channel.
+<2> `--sink` specifies the target destination to which the event should be delivered. By default, the `<sink_name>` is interpreted as a Knative service of this name, in the same namespace as the subscription. You can specify the type of the sink by using one of the following prefixes:
+`ksvc`:: A Knative service.
+`svc`:: A Kubernetes Service.
```

Comment:
`kn` doesn't support `svc` prefix directly, but it can be achieved by using. I believe that was intentional decision to not confuse users with `ksvc` and `svc`. 
However, the desired behaviour can be achieve by providing URI, e.g.:
```
 '--sink https://event.receiver.uri' for an URI with an 'http://' or 'https://' schema
```

Reply:
I think I'll just remove `svc` for now then if that's OK since I want to keep the core focus here on eventing stuff, but I believe there's a Jira in the backlog to improve the docs for services as a whole and one to document updating a service using kn, so maybe we can cover this in more detail when we revisit those docs and things like expanding CLI reference and config docs?

Reply:
Sure, fine by me. Removing `svc` is definitely sufficient here as it'd produce an error message that it isn't supported. The example above is copied from the flag's help message, so there's a way to find it out. :)